main.py

```diff
@@ -2,6 +2,37 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    for item_first, item_second in actors:
+        Actor.objects.create(
+            first_name=item_first,
+            last_name=item_second
+        )
```

Comment:
I think u can change it to generator, but it's up to you

Reply:
Also for best readability you can change item_first, item_second to first_name, last_name

Reply:
@Kirontiko is right, it is better to name variables as first_name, last_name for better readability